api/v01/entities/vrp_input.rb

```diff
@@ -0,0 +1,522 @@
+    optional(:overall_duration, type: Integer, values: ->(v) { v.positive? }, documentation: { hidden: true }, desc: '(Scheduling only) If schedule covers several days, maximum work duration over whole period. Not available with periodic heuristic.', coerce_with: ->(value) { ScheduleType.type_cast(value) })
+    optional(:distance, type: Integer, desc: 'Maximum tour distance. Not available with periodic heuristic.')
+    optional(:maximum_ride_time, type: Integer, desc: 'Maximum ride duration between two route activities')
+    optional(:maximum_ride_distance, type: Integer, desc: 'Maximum ride distance between two route activities')
+    optional :skills, type: Array[Array[String]], desc: 'Particular abilities which could be handle by the vehicle. This parameter is a set of alternative skills, and must be defined as an Array[Array[String]]. Not available with periodic heuristic.',
+                      coerce_with: ->(val) { val.is_a?(String) ? [val.split(/,/).map(&:strip)] : val } # TODO : Create custom coerce to consider multiple alternatives
+
+    optional(:unavailable_work_day_indices, type: Array[Integer], desc: '(Scheduling only) Express the exceptionnals indices of unavailabilty')
+    optional(:unavailable_work_date, type: Array, desc: '(Scheduling only) Express the exceptionnals days of unavailability')
+    mutually_exclusive :unavailable_work_day_indices, :unavailable_work_date
+
+    optional(:free_approach, type: Boolean, desc: 'Do not take into account the route leaving the depot in the objective. Not available with periodic heuristic.')
+    optional(:free_return, type: Boolean, desc: 'Do not take into account the route arriving at the depot in the objective. Not available with periodic heuristic.')
+  end
+
+  params :vehicle_costs do
+    optional(:cost_fixed, type: Float, desc: 'Cost applied if the vehicle is used')
+    optional(:cost_distance_multiplier, type: Float, desc: 'Cost applied to the distance performed')
+
+    optional(:cost_time_multiplier, type: Float, desc: 'Cost applied to the total amount of time of travel (Jsprit) or to the total time of route (ORtools). Not taken into account within periodic heuristic.')
+    optional(:cost_value_multiplier, type: Float, desc: 'Multiplier applied to the value matrix and additional activity value. Not taken into account within periodic heuristic.')
+    optional(:cost_waiting_time_multiplier, type: Float, desc: 'Cost applied to the waiting time in the route. Not taken into account within periodic heuristic.')
+    optional(:cost_late_multiplier, type: Float, desc: 'Cost applied if a point is delivered late (ORtools only). Not taken into account within periodic heuristic.')
+    optional(:cost_setup_time_multiplier, type: Float, desc: 'Cost applied on the setup duration (Jsprit only). Not taken into account within periodic heuristic.')
+  end
+
+  params :vehicle_model_related do
```

Comment:
Why is this specific ?

Reply:
These objects are related to other objects of the model and will be interpreted by the model create